src/classes/UrlHelper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Cronbeat;
+
+class UrlHelper {
+    public static function parseControllerFromUrl(): string {
+        $uri = $_SERVER['REQUEST_URI'];
+        $uri = trim($uri, '/');
+        $uri = explode('/', $uri);
+        return ($uri[0] !== '') ? $uri[0] : 'login';
+    }
+    
+    public static function parsePathWithoutController(): string {
+        $uri = $_SERVER['REQUEST_URI'];
+        $uri = trim($uri, '/');
+        $uri = explode('/', $uri);
+        
+        if (count($uri) > 1) {
+            return implode('/', array_slice($uri, 1));
+        }
+        
+        return '';
+    }
```

Comment:
@jetbrains-junie the first 3 lines of this function are identical to parseControllerFromUrl, extract a function out of it